### ISSUE TITLE
Prevent syncing from deleting text I've added to a note through Obsidian.

### ISSUE THREAD

**Issue by Aleex-C** [omnivore-app/obsidian-omnivore#249]
Is there a way to keep text added through Obsidian in an Omnivore note and not be deleted when performing a "Fetch Articles"? I sometimes take notes on some articles directly in Obsidian and then remember I have to paste them in the Omnivore Notes so as not to get deleted (overwritten) after sync. Wondering if anyone encountered this "issue" and what workaround they found? Thanks.

**Comment by darthwalsh** [comment]
Yeah that's an interesting question. there's plenty of discussion in #45 -- maybe that would solve this problem?
